node/src/cli.rs

```diff
@@ -20,57 +25,59 @@ pub struct Cli {
 
 
 /// Possible subcommands of the main binary.
-#[derive(Debug, clap::Subcommand)]
+#[derive(Debug, Parser)]
```

Comment:
Assuming this is fine? why clap::Subcommand is replaced?

Reply:
Yes